lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/table/DBOColumnModel.java

```diff
@@ -116,7 +121,20 @@ public MigrationType getMigratableTableType() {
 
 	@Override
 	public MigratableTableTranslation<DBOColumnModel, DBOColumnModel> getTranslator() {
-		return new BasicMigratableTableTranslation<DBOColumnModel>();
+		return new BasicMigratableTableTranslation<DBOColumnModel>(){
+			@TemporaryCode(author="ziming", comment = "one-time migration change. remove after stack 309")
+			@Override
+			public DBOColumnModel createDatabaseObjectFromBackup(DBOColumnModel backup) {
+				//doing this round trip will assign default value to maxListLength
+				ColumnModel columnModel = ColumnModelUtils.createDTOFromDBO(backup);

```

Comment:
let  ColumnModelUtils.createDTOFromDBO(backup); handle the set to default (not setting it here)